README.adoc

```diff
@@ -208,20 +237,45 @@ mvn pact:publish
 After the file is published, you'll see a similar output to the following example:
 [role="no_copy"]
 ```
---- maven:4.1.0:publish (default-cli) @ inventory ---
+--- maven:4.1.21:publish (default-cli) @ inventory ---
 Publishing 'Inventory-System.json' with tags 'open-liberty-pact' ... OK
 ```
 
 == Verifying the pact in the Pact Broker
 
-Refresh the Pact Broker webpage at the http://localhost:9292/[^] URL to verify that a new entry exists. There isn't yet a
-timestamp in the last verified column because the pact hasn't been verified by the `system` microservice.
+// Static guide instruction
+ifndef::cloud-hosted[]
+Refresh the Pact Broker webpage at the http://localhost:9292/[^] URL to verify that a new entry exists.
+There isn't yet a timestamp in the last verified column because the pact hasn't been verified
+by the `system` microservice.
+endif::[]
+
+// Cloud hosted guide instruction
+ifdef::cloud-hosted[]
+Refresh the Pact Broker at the **`https://accountname-9292.theiadocker-4.proxy.cognitiveclass.ai`** URL, 
+where **accountname** is your account name.
+There isn't yet a timestamp in the last verified column because the pact hasn't been verified
+by the `system` microservice.
+endif::[]
 
 image::pact-broker-webpage-refresh.png[Pact Broker webpage for new entry, align="left"]
 
 {empty} +
 
-You can see detailed insights about each interaction by going to the http://localhost:9292/pacts/provider/System/consumer/Inventory/latest[^] URL, as shown in the following image:
+// Static guide instruction
+ifndef::cloud-hosted[]
+You can see detailed insights about each interaction by going to the
+http://localhost:9292/pacts/provider/System/consumer/Inventory/latest[^] URL,
+as shown in the following image:
+endif::[]
+
+// Cloud hosted guide instruction
+ifdef::cloud-hosted[]
+You can see detailed insights about each interaction by going to the
+**`https://accountname-9292.theiadocker-4.proxy.cognitiveclass.ai/pacts/provider/System/consumer/Inventory/latest`** URL, 
+where **accountname** is your account name.
+It will appear similar to the following image:
```

Comment:
It will appear similar to the following image:

--->

The insights look similar to the following image: